app/src/main/java/com/gh4a/Gh4Application.java

```diff
@@ -34,13 +34,17 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatDelegate;
 import androidx.multidex.MultiDexApplication;
+import androidx.work.Configuration.Provider;
 
 /**
  * The Class Gh4Application.
  */
-public class Gh4Application extends MultiDexApplication implements OnSharedPreferenceChangeListener {
+public class Gh4Application extends MultiDexApplication implements
+        Provider,
```

Comment:
Nit: I think it would be better to make this `androidx.work.Configuration.Provider` here and nuke the import to make it more clear what this is about.

Reply:
Sure, done :)